Replaced /500 redirect with <Error500 />

ojbravo opened the pull request:
Closes #864

billhimmelsbach commented on e2e/pages/filing-app/unavailableApis.spec.ts:
```diff
@@ -0,0 +1,208 @@
+
+    // Unblock API Call
+    await test.step('Unblock API', async () => {
+      await blockApi(page, new RegExp(`.*?/v1/institutions/${testLei}`), false);
```
Same comment as above: To let us punt on adding the variables to the fixture for a follow-up PR, we can adjust this RegEx to be something that will catch test LEIs generally: `.*?/v1/institutions/.*TESTACCT.*` or we can just simulate the entire FI API being down for this part of the test with `.*?/v1/institutions/.*` which might be more realistic?

ojbravo replied:
Update made.

billhimmelsbach commented on e2e/utils/blockApi.ts:
```diff
@@ -0,0 +1,20 @@
+import type { Page } from '@playwright/test';
```
Great!

billhimmelsbach commented on e2e/pages/filing-app/unavailableApis.spec.ts:
```diff
@@ -0,0 +1,208 @@
+
+    // Confirm Error Alert
+    await test.step('Error Alert', async () => {
+      test.setTimeout(120_000);
```
Yeah, the curse/blessing of having a bunch of retries. These tests are really great, but it will be a bummer having long waits if we end up running these tests often in CI. Can we make that ticket to make retries a variable affected by the environment, so when we're testing we prevent or limit retries? 

Then maybe let's add a comment above these long retries with a link to the ticket ([like this one](https://github.com/cfpb/sbl-frontend/blob/main/src/components/BetaAndLegalNotice.tsx#L27-L28)) to remind ourselves to adjust these timeouts when we're done?

ojbravo replied:
Created issue #916 and added ToDo comments.

billhimmelsbach reviewed (changes requested):
These tests are really awesome @ojbravo! I have a few comments, but generally, this is a huge help to the robustness of the test suite!

ojbravo commented on e2e/fixtures/testFixture.ts:
```diff
@@ -8,6 +8,9 @@ import createInstitution from '../utils/createInstitution';
 import getAdminKeycloakToken from '../utils/getKeycloakToken';
 
+// eslint-disable-next-line import/no-mutable-exports
+let testLei: string;
```
I'll use the regex to look for 'TESTACCT053'.

ojbravo commented on e2e/pages/filing-app/unavailableApis.spec.ts:
```diff
@@ -0,0 +1,208 @@
+
+    // Block API Call: **/v1/institutions/
+    await test.step('Block API: /v1/institutions', async () => {
+      await blockApi(page, new RegExp(`.*?/v1/institutions/${testLei}`), true);
```
Good call. Updated.

ojbravo commented:
Latest updates:
1. Correctly used testfixture at beginning of test
2. Reverted to original testfixtures.ts and modified regex instead
3. Added uploadFile.ts from #881 
4. Updated Playwright Actions labels
5. Created issue #916 and added to ToDo comments.
